testng-core/src/test/java/org/testng/xml/internal/TestNamesMatcherTest.java

```diff
@@ -65,7 +65,7 @@ public void testCloneIfContainsTestsWithNamesMatchingAnyNegativeCondition(
       testCloneIfContainsTestsWithNamesMatchingAnyNegativeConditionWithIgnoreMissedTestNamesEnabled(
           XmlSuite xmlSuite, List<String> names) {
     boolean ignoreMissedTestNames = true;
-    TestNamesMatcher testNamesHelper = new TestNamesMatcher(xmlSuite, names, ignoreMissedTestNames);
+    new TestNamesMatcher(xmlSuite, names, ignoreMissedTestNames);
```

Comment:
Useless line

Reply:
Same case here. The test is vetting out the exception that can be thrown by the constructor of `TestNamesMatcher`